src/main/sarl/fr/utbm/info/vi51/worldswar/simulator/Simulator.sarl

```diff
@@ -35,7 +35,7 @@ agent Simulator {
 	private var listeners : List<SimulatorListener>
 
 	/** Simulation speed, as chosen by the user */
-	private var simulationSpeed = SimulationSpeed.NORMAL
+	private var simulationSpeed = SimulationSpeed.getDefaultSpeed
```

Comment:
Initial speed used when the simulator is initialized. When a new simulation is started, the speed of the previous simulation is kept for the new one